gradlew

```diff
@@ -1,25 +1,43 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
 
 ##############################################################################
 ##
 ##  Gradle start up script for UN*X
 ##
 ##############################################################################
 
-# Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+# Attempt to set APP_HOME
+# Resolve links: $0 may be a link
+PRG="$0"
```

Comment:
What does PRG means? Is it "program", the shell script to execute? I suggest maybe to use another name.

Reply:
this file is generated by the gradle wrapper, we should not touch the generated file, even if it's confusing